### PR TITLE
Added Flex attributes

### PR DESCRIPTION
**Pull request by MiskaVivolin**

### CODE REVIEW

File: entities/style/StyleEntity.ts [hunk 1]
```diff
@@ -107,7 +114,17 @@ export const StyleEntityFactory = (
...
                      .add( EntityFactoryImpl.createProperty("boxSizing").setTypes(BoxSizing, VariableType.UNDEFINED) )
-);
+                     .add( EntityFactoryImpl.createProperty("display").setTypes(Display, VariableType.UNDEFINED) )
+                     .add( EntityFactoryImpl.createProperty("flexDirection").setTypes(FlexDirection, VariableType.UNDEFINED) )
+                     .add( EntityFactoryImpl.createProperty("flexWrap").setTypes(FlexWrap, VariableType.UNDEFINED) )
+                     .add( EntityFactoryImpl.createProperty("justifyContent").setTypes(JustifyContent, VariableType.UNDEFINED) )
+                     .add( EntityFactoryImpl.createProperty("alignItems").setTypes(AlignItems, VariableType.UNDEFINED) )
+                     .add( EntityFactoryImpl.createProperty("alignContent").setTypes(AlignContent, VariableType.UNDEFINED) )
+                     .add( EntityFactoryImpl.createProperty("alignSelf").setTypes(AlignSelf, VariableType.UNDEFINED) )
+                     .add( EntityFactoryImpl.createProperty("flexShrink").setTypes(VariableType.NUMBER) )
```
Comment (thejhh) [inline entities/style/StyleEntity.ts:124]: Missing `VariableType.UNDEFINED` from these number fields

File: entities/types/Display.ts [hunk 2]
```diff
@@ -0,0 +1,31 @@
+import { isUndefined } from "lodash";
+import { isEnum, explainEnum, stringifyEnum, parseEnum } from "../../types/Enum";
+import { explainOk, explainNot, explainOr } from "../../types/explain";
+
+export enum Display {
+    FLEX = 'flex'
```
Comment (thejhh) [inline entities/types/Display.ts:6]: This should have the other standard types, see https://developer.mozilla.org/en-US/docs/Web/CSS/display

**Review by thejhh — CHANGES_REQUESTED** [review]
Looks fine, few changes requested

**Comment by MiskaVivolin** [comment]
resolved requests

**Review by thejhh — APPROVED** [review]
Looks good

**Pull request merged**